postgres/datadog_checks/postgres/statements.py

```diff
@@ -333,11 +343,33 @@ def _emit_pg_stat_statements_metrics(self):
             )
         except psycopg2.Error as e:
             self._log.warning("Failed to query for pg_stat_statements count: %s", e)
+    
+    def _emit_pg_stat_statement_query_text_length_metrics(self, rows):
+        txt_length_sum = 0
+        txt_length_max = 0
+        for r in rows:
+            query_length = len(r['query'])
+            txt_length_sum += query_length
+            if query_length > txt_length_max:
+                txt_length_max = query_length
+        self._check.gauge(
```

Comment:
could we compute these numbers while we iterate over the rows from this method https://github.com/DataDog/integrations-core/pull/13433/files#diff-157c2960ae5c7408275369d11207e5635cb0515cdc0b898396c4d043df08576dR374 or is that a pain?

Reply:
That's a good point. If we also change this to a histogram, I think this could be one call per row in the existing iteration over rows: 
```
self._check.histogram("postgresql.pg_stat_statements.query_text_length", len(r['query']), ...

Reply:
Oh nice! -- ;shame-metricsnoob;